client/src/lib/helper.test.js

```diff
@@ -0,0 +1,14 @@
+jest.mock('jquery');
+
+const helper = require('./helper');
+
+describe('client/src/lib/helper', () => {
+  test('should blah', () => {
```

Comment:
I know your commit says what you were testing for but just nitpicking at the fact that inside the test it's just blah

Reply:
good catch, will update to correctly describe the test